### PR TITLE
Front end design work (Home page, Header, Navbar, Footer, Login, Orders, Products)

### PR DESCRIPTION
**Pull request by dennisccastro**

### CODE REVIEW

**Review by frankjoesilva — COMMENTED** [review]
looks good so far Dennis, just wanted to double check if you were able to test all the code locally before pushing up?

File: src/components/Allproducts.js [hunk 1]
```diff
@@ -20,15 +22,20 @@ export default function Allproducts() {
   });
 
   return (
-    <div>
+    <div className='allproducts'>
```
Comment (frankjoesilva) [inline src/components/Allproducts.js:25]: Let's put a dash in allproducts for readability.

File: package.json [hunk 2]
```diff
@@ -18,11 +18,12 @@
...
     "express": "^4.17.1",
     "morgan": "^1.10.0",
+    "node": "^15.2.1",
     "pg": "^8.5.1",
     "react": "^16.14.0",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.1"
+    "react-scripts": "^4.0.0"
   },
```
Comment (frankjoesilva) [inline package.json:27]: is the 4.0.0 for the react-scripts just the updated version?

Comment (dennisccastro) [reply package.json:27]: Yes, I did an audit fix.

File: src/components/Allproducts.js (hunk 1 above)
Comment (dennisccastro) [reply src/components/Allproducts.js:25]: Will do so on the second pass.

**Comment by dennisccastro** [comment]
> looks good so far Dennis, just wanted to double check if you were able to test all the code locally before pushing up?

Yes, I did. It was working locally.

**Review by frankjoesilva — APPROVED** [review]

**Pull request merged**